### DISCUSSION TITLE
Combined_res question

### DISCUSSION THREAD

**Discussion by daacarri** [egeulgen/pathfindR#102] [Q&A] [answered]
In the vignette a data frame called combined_res is made by combining itself with current_res, right before visualization. However it’s not clear on what is being combined as that data frame is not made anywhere else in the code. Is it a typo and are you suppose to combine with the data frame made right before current_res? I tried and got a character length error. I also tried with just the current res df and got results but only 4 genes contributed. Not sure if that’s a typo in the vignette or what.

**Comment by egeulgen** [comment, marked as answer]
I'm assuming you meant the [Step-by-Step Execution vignette](https://egeulgen.github.io/pathfindR/articles/manual_execution.html).

Have you encountered any issues with `run_pathfindR()`? We generally suggest using this wrapper function instead of manually performing the steps.

In the vignette, the `combined_res` dataframe is used to store the result of each iteration and is created right before the loop (`combined_res <- NULL`). Within the loop, each iteration's result (stored in `current_res`) is added to the `combined_res`. 

Let me know if this helps,
Best,
-E

**Reply by daacarri** [reply]
I have it now, thanks. However, it only returned 4 distinct genes (and their subsequent pathways). What are the odds only 4 genes/pathways would be returned?

**Reply by egeulgen** [reply]
Depending on the size of your input genes, yes it is possible
